src/actions/form-template-item-actions.js

```diff
@@ -0,0 +1,397 @@
+
+export const getFormTemplateItems =
+  (
+    formTemplateId,
+    term = null,
+    page = DEFAULT_CURRENT_PAGE,
+    perPage = DEFAULT_PER_PAGE,
+    order = "id",
+    orderDir = DEFAULT_ORDER_DIR
+  ) =>
+  async (dispatch) => {
+    const accessToken = await getAccessTokenSafely();
+    const filter = [];
+
+    dispatch(startLoading());
+
+    if (term) {
+      const escapedTerm = escapeFilterValue(term);
+      filter.push(`name=@${escapedTerm},code=@${escapedTerm}`);
+    }
+
+    const params = {
+      page,
+      fields: "id,code,name",
+      per_page: perPage,
+      access_token: accessToken
+    };
+
+    if (filter.length > 0) {
+      params["filter[]"] = filter;
+    }
+
+    // order
+    if (order != null && orderDir != null) {
+      const orderDirSign = orderDir === 1 ? "" : "-";
+      params.ordering = `${orderDirSign}${order}`;
+    }
+
+    return getRequest(
+      createAction(REQUEST_FORM_TEMPLATE_ITEMS),
+      createAction(RECEIVE_FORM_TEMPLATE_ITEMS),
+      `${window.INVENTORY_API_BASE_URL}/api/v1/form-templates/${formTemplateId}/items/`,
+      authErrorHandler,
+      { order, orderDir, page, term }
+    )(params)(dispatch).then(() => {
+      dispatch(stopLoading());
+    });
+  };
+
+export const getFormTemplateItem =
+  (formTemplateId, formTemplateItemId) => async (dispatch) => {
+    const accessToken = await getAccessTokenSafely();
+
+    dispatch(startLoading());
+
+    const params = {
+      access_token: accessToken,
+      expand: "images,meta_fields,meta_fields.values"
+    };
+
+    return getRequest(
+      null,
+      createAction(RECEIVE_FORM_TEMPLATE_ITEM),
+      `${window.INVENTORY_API_BASE_URL}/api/v1/form-templates/${formTemplateId}/items/${formTemplateItemId}/`,
+      authErrorHandler
+    )(params)(dispatch).then(() => {
+      dispatch(stopLoading());
+    });
+  };
+
+export const deleteFormTemplateItem =
+  (formTemplateId, formTemplateItemId) => async (dispatch) => {
+    const accessToken = await getAccessTokenSafely();
+
+    dispatch(startLoading());
+
+    const params = {
+      access_token: accessToken
+    };
+
+    return deleteRequest(
+      null,
+      createAction(FORM_TEMPLATE_ITEM_DELETED)({
+        formTemplateId,
+        formTemplateItemId
+      }),
+      `${window.INVENTORY_API_BASE_URL}/api/v1/form-templates/${formTemplateId}/items/${formTemplateItemId}/`,
+      null,
+      authErrorHandler
+    )(params)(dispatch).then(() => {
+      dispatch(stopLoading());
+    });
+  };
+
+export const resetFormTemplateItemForm = () => (dispatch) => {
+  dispatch(createAction(RESET_FORM_TEMPLATE_ITEM_FORM)({}));
+};
+
+const normalizeEntity = (entity) => {
+  const normalizedEntity = { ...entity };
+  normalizedEntity.meta_fields = normalizedEntity.meta_fields?.filter(
+    (mf) => mf.name
+  );
+  normalizedEntity.images = normalizedEntity.images?.filter(
+    (img) => img.file_path
+  );
+  normalizedEntity.early_bird_rate = amountToCents(
```

Comment:
@romanetar are these *_rate fields mandatory ?
if not we should check if those are set first in order to avoid errors